### PR TITLE
#25: Added some docs

### PR DESCRIPTION
**Pull request by KnYaZuK**

### CODE REVIEW

File: lib/src/components/action.dart [hunk 1]
```diff
@@ -44,6 +61,16 @@ abstract class Action<T extends GlobalState> extends ReduxAction<T> {
     };
   }
 
+  /// This is an optional method that may be overridden to run during action
+  /// dispatching, after `reduce`. If this method throws an error, the
+  /// error will be swallowed (will not throw). So you should only run code that
+  /// can't throw errors. It may be synchronous only.
+  /// Note this method will always be called,
+  /// even if errors were thrown by `before` or `reduce`.
```
Comment (w3ggy) [inline lib/src/components/action.dart:69]: Maybe better to add a link to super.before doc instead? What do you think?

I mean in all such methods

File: lib/src/states/operation_state.dart [hunk 2]
```diff
@@ -7,13 +10,18 @@ enum OperationState {
 }
 
 extension OperationStateExtension on OperationState {
+  /// Returns `true` if [OperationState.idle]
```
Comment (w3ggy) [inline lib/src/states/operation_state.dart:13]:
```suggestion
  /// Returns `true` if the operation state is equals to [OperationState.idle]
```

**Comment by w3ggy** [comment]
What about version changing?

File: lib/src/components/action.dart (hunk 1 above)
Comment (KnYaZuK) [reply lib/src/components/action.dart:69]: This method already has the "copy" from the parent docs. If we wanna to add some new tips - we should copy from parent too, or not?

**Comment by KnYaZuK** [comment]
Incremented version to 3.4.1 and fixed the review issues

**Review by w3ggy — APPROVED** [review]

**Pull request merged**